includes/class-wc-emails.php

```diff
@@ -611,6 +611,16 @@ public function low_stock( $product ) {
 			return;
 		}
 
+		/**
+		 * Action hook fired to determine if the current product should trigger a
```

Comment:
It doesn't sounds accurate, also it's very specif, better change it to something more generic, so makes sense for other user cases too.